test/molecule-role/molecule/default/tests/test_agent.py

```diff
@@ -42,10 +42,18 @@ def wait_for_check_successes():
     agent_log = host.file(agent_log_path).content_string
 
     # Check for errors
+    # count = 0
     for line in agent_log.splitlines():
         print("Considering: %s" % line)
         # TODO: Update event endpoint should get rid of this error,
         # once STAC-2500 is fixed
+        if re.search(
+                "Error code \"400 Bad Request\" received while sending " +
+                "transaction to \"https://.*/stsAgent/intake/",
+                line):
+            continue
+
+        # Before restarting the logs contain error for previous sts_url
         if re.search(
```

Comment:
We will never see http again in the logs, why keeping this assertion ?

Reply:
before changing the URL and SSL flag in the conf file and restarting agent, the log contains `http` log.